src/dune_engine/package.ml

```diff
@@ -225,33 +225,94 @@ module Dependency = struct
 end
 
 module Source_kind = struct
+  module Host = struct
+    type kind =
+      | Github
+      | Bitbucket
+      | Gitlab
+
+    let kind_string_map =
+      [ (Github, "github"); (Bitbucket, "bitbucket"); (Gitlab, "gitlab") ]
```

Comment:
This is only used as `List.assoc kind_string_map kind |> Option.value_exn`. Feels like a plain `function Github -> "github" | ...` function would be simpler and more efficient